shell/platform/windows/compositor.h

```diff
@@ -19,6 +21,7 @@ namespace flutter {
 // Platform views are not yet supported.
 class Compositor {
  public:
+  Compositor(PlatformViewManager* manager) : platform_view_manager_(manager) {}
```

Comment:
`Compositor` is just an interface that knows nothing about the implementation. I'd remove the `platform_view_manager_` field from this type and copy it in both the opengl and software compositors